cnxepub/formatters.py

```diff
@@ -95,15 +95,15 @@ def _generate_ids(self, document, content):
             'blockquote', 'q', 'code', 'pre', 'object', 'img', 'audio',
             'video',
             ]
-        elements_xpath = '|'.join(['.//{}|xhtml:{}'.format(elem, elem)
+        elements_xpath = '|'.join(['//{}|//xhtml:{}'.format(elem, elem)
```

Comment:
The top level element is a temporary wrapper because etree requires a root element.  It gets removed when we change the content to string again.  I think we should use `.//`.